bin/cylc-submit

```diff
@@ -101,8 +103,11 @@ def main():
 
     # Initialise job submit environment
     glbl_cfg().create_cylc_run_tree(suite)
-    task_job_mgr = TaskJobManager(
-        suite, SuiteProcPool(), SuiteDatabaseManager(), suite_srv_mgr)
+    pool = SuiteProcPool()
+    db_mgr = SuiteDatabaseManager()
+    b_mgr = BroadcastMgr(db_mgr)
+    te_mgr = TaskEventsManager(suite, pool, db_mgr, b_mgr)
+    task_job_mgr = TaskJobManager(suite, pool, db_mgr, suite_srv_mgr, te_mgr)
```

Comment:
Is this change still necessary?

Reply:
What do you mean exactly by "this change"?  Oh, maybe just that the `te_mgr` and `b_bgr` variables are unnecessary as only used once?   I'll change that...

Reply:
Done.